components/centraldashboard/public/components/dashboard-view.js

```diff
@@ -55,16 +79,42 @@ export class DashboardView extends PolymerElement {
                 value: [
                     {
                         text: 'Open docs',
-                        link: `${kubeflowDocs}/getting-started/`,
+                        link: `${DOCS}/getting-started/`,
                     },
                     {
                         text: 'Open Github',
                         link: 'https://github.com/kubeflow/kubeflow',
                     },
                 ],
             },
+            platformName: String,
+            platformLinks: Array,
+            platformInfo: {
+                type: Object,
+                observer: '_platformInfoChanged',
+            },
         };
     }
+
+    /**
+     * Observer for platformInfo property
+     */
+    _platformInfoChanged() {
+        if (this.platformInfo && this.platformInfo.providerName === 'gce') {
+            this.platformName = 'GCP';
+            const pieces = this.platformInfo.provider.split('/');
+            let gcpProject = '';
+            if (pieces.length >= 3) {
+                gcpProject = pieces[2];
+            }
+            this.platformLinks = GCP_LINKS.map((l) => {
+                return {
+                    text: l.text,
+                    link: l.link + gcpProject,
+                };
+            });
```

Comment:
This chunk above can be written as:
```suggestion
            this.platformLinks = GCP_LINKS.map((l) => ({
                text: l.text,
                link: l.link + gcpProject,
            }));
```

Reply:
Understood but it does require the additional enclosing parentheses to tell the interpreter to treat the expression as an object literal. I wish it worked a little more like single-statement lambdas in Java but for multi-line expressions, I find the explicit return to be a bit more obvious when reading.

Reply:
Yeah, all languages have their nuances. But if you think `return`s are generally more readable, we can stick to that type of syntax.